docs/sources/api-response-examples/atlassian/jira/sanitized/issue_v2.json

```diff
@@ -1,182 +1,99 @@
           "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
+          "accountType": "atlassian",
           "active": true,
-          "timeZone": "America/New_York",
-          "accountType": "atlassian"
+          "timeZone": "America/New_York"
         },
         "created": "2023-06-09T12:31:44.105-0400",
-        "size": 2829492,
-        "mimeType": "audio/mpeg",
-        "content": "https://api.atlassian.com/ex/jira/f6eef702-e05d-43ba-bd5c-75fce47d560e/rest/api/2/attachment/content/10000"
+        "id": "10000"
       }
     ],
-    "summary": "worklog test",
-    "creator": {
-      "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
-      "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
-      "active": true,
-      "timeZone": "America/New_York",
-      "accountType": "atlassian"
-    },
-    "subtasks": [],
-    "reporter": {
-      "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
-      "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
-      "active": true,
-      "timeZone": "America/New_York",
-      "accountType": "atlassian"
-    },
-    "aggregateprogress": {
-      "progress": 28800,
-      "total": 518400,
-      "percent": 5
-    },
-    "customfield_10000": null,
-    "customfield_10001": null,
-    "customfield_10002": "0|i0003j:",
-    "customfield_10200": {
-      "hasEpicLinkFieldDependency": false,
-      "showField": false,
-      "nonEditableReason": {
-        "reason": "PLUGIN_LICENSE_ERROR",
-        "message": "The Parent Link is only available to Jira Premium users."
-      }
-    },
-    "customfield_10003": null,
-    "customfield_10004": null,
-    "customfield_10400": null,
-    "customfield_10710": null,
-    "customfield_10711": null,
-    "environment": null,
-    "customfield_10712": null,
-    "customfield_10713": null,
-    "customfield_10714": null,
+    "resolutiondate": null,
     "duedate": null,
-    "customfield_10715": null,
-    "customfield_10716": null,
-    "customfield_10717": null,
-    "customfield_10718": null,
     "progress": {
-      "progress": 28800,
       "total": 518400,
-      "percent": 5
+      "progress": 28800
     },
+    "issuelinks": [],
     "votes": {
-      "votes": 0,
-      "hasVoted": false
+      "hasVoted": false,
+      "votes": 0
     },
-    "customfield_10719": null,
+    "assignee": null,
     "worklog": {
-      "startAt": 0,
-      "maxResults": 20,
-      "total": 1,
       "worklogs": [
         {
+          "issueId": "10711",
+          "timeSpentSeconds": 28800,
+          "timeSpent": "1d",
           "author": {
-            "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
```

Comment:
if we *can* get this, would be good; just to have more data ponts linking `accountId` --> `emailAddress`

Reply:
Ouch, fixed. Thanks.